shared/src/ledger/eth_bridge/vp/mod.rs

```diff
@@ -350,32 +439,91 @@ mod tests {
             .expect("should always be able to construct this key")
     }
 
+    /// Initialize some dummy storage for testing
+    fn setup_storage() -> Storage<MockDB, Sha256Hasher> {
+        let mut storage = Storage::<MockDB, Sha256Hasher>::open(
+            std::path::Path::new(""),
+            ChainId::default(),
+            None,
+        );
+
+        // setup a user with a balance
+        let balance_key = balance_key(
+            &xan(),
+            &Address::decode(ARBITRARY_OWNER_A_ADDRESS).expect("Test failed"),
+        );
+        storage
+            .write(
+                &balance_key,
+                Amount::from(100).try_to_vec().expect("Test failed"),
```

Comment:
let's replace the `100`s and `0`s in this fn and in the tests with consts, something that will link the values between the tests e.g. `ARBITRARY_OWNER_A_INITIAL_BALANCE`, `BRIDGE_POOL_ESCROW_INITIAL_BALANCE`